packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -547,9 +555,14 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
         );
     }
 
+    // NOTE: Search blob concept
+    public registerExtraSnapshotContracts(contracts: ISnapshotContracts) {
```

Comment:
Based on the signature of this.extraSnapshotContracts, I'd assume this method should have this signature:
```
public registerExtraSnapshotContracts(contractName: string, extraBlobFn: () => string)
```